ldm/invoke/restoration/gfpgan.py

```diff
@@ -3,25 +3,29 @@
 import os
 import sys
 import numpy as np
+from ldm.invoke.globals import Globals
 
 from PIL import Image
 
 
 class GFPGAN():
     def __init__(
             self,
-            gfpgan_model_path='./models/gfpgan/GFPGANv1.4.pth') -> None:
-
-        self.model_path = os.path.join(gfpgan_model_path)
+            gfpgan_model_path='models/gfpgan/GFPGANv1.4.pth'
+    ) -> None:
+    
+        if not os.path.isabs(gfpgan_model_path):
+            gfpgan_model_path=os.path.join(Globals.root,gfpgan_model_path)
```

Comment:
Does this need either `normpath` or `abspath`?

Reply:
Added `abspath()` as suggested. This shouldn't be necessary, but the path does get handed off to a third-party function call and it could do something special with relative pathnames, you never know.